### PR TITLE
UX Pain Points

### PR DESCRIPTION
**Pull request by klohman**
ux pain points - trello cards

### CODE REVIEW

File: web-api/storage/fixtures/seed/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module.exports = [
+  /*
```
Comment (rachaelparris) [inline web-api/storage/fixtures/seed/index.js]: Could you uncomment this? It was just a local change that doesn't need to be committed.

File: web-client/src/styles/custom.scss [hunk 2]
```diff
@@ -252,7 +252,7 @@ select[disabled] {
 }
 
 .required-statement  {
-  font-weight: $font-semibold;
+  font-weight: 600;
```
Comment (rachaelparris) [inline web-client/src/styles/custom.scss:255]: Is there a reason to not use the `$font-semibold` variable here?

Comment (klohman) [reply web-client/src/styles/custom.scss:255]: I tried it with $font-semibold and it worked locally, but not on dev for some reason. Just trying another option.

File: web-api/storage/fixtures/seed/index.js (hunk 1 above)
Comment (klohman) [reply web-api/storage/fixtures/seed/index.js]: Done!

File: web-client/src/styles/buttons.scss [hunk 3]
```diff
@@ -92,8 +92,9 @@
 }
 
 .button-switch-box {
-  font-size: 14px;
-  font-weight: $font-semibold;
+  font-size: 17px;
+  font-weight: $font-normal;
+  font-family: 'Source Sans Pro';
```
Comment (rachaelparris) [inline web-client/src/styles/buttons.scss]: You'll need to use the variable here - `$font-source-sans` - and then try running `npm run lint:fix` to fix your lint errors.

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/styles/custom.scss (hunk 2 above)
Comment (kkoskelin) [reply web-client/src/styles/custom.scss:255]: Once compiled (and assuming no cache issue), the font variable ought to be working.  If not, I'd be very much interested in a demo of it _not_ working so we can determine if there's a compilation issue.

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**